src/Symfony/Component/Console/Tests/Helper/ProcessHelperTest.php

```diff
@@ -44,6 +44,9 @@ public function testPassedCallbackIsExecuted()
         $this->assertTrue($executed);
     }
 
+    /**
+     * @return array
+     */
     public function provideCommandsAndOutput()
```

Comment:
Your IDE can already determine the return type, in this state it doesn't really add a value